### ISSUE TITLE
update the doc link to deploy a nuxt application (Read.me)

### ISSUE THREAD

**Issue by Kalo0m** [viandwi24/nuxt3-awesome-starter#15]
the link in the read.me is invalid

**Comment by viandwi24** [comment]
I'll fix that later, or could you have some time to make a pull request?

**Comment by Kalo0m** [comment]
Yes I'll do it today, It'll be my first open source contribution ahah !

Have a nice day.

Théo

**Comment by Kalo0m** [comment]
I can’t push to my branch, I don’t have permissions. Is it normal ?

**Comment by viandwi24** [comment]
For pull request, you must fork this repo first to your account. 

only then can you clone it to your computer and then try to create a new branch if necessary, or stay on the main branch. 

after that start working on it. once done as usual, you can commit and then push to your repo.

Finally, enter the pull request menu in this repo, then enter your repo as a reference for pull requests, add information and so on.

**Comment by viandwi24** [comment]
If there is a permission error, make sure that you push to your own repo (after fork). Don't push to this repo, because of course you don't have permission.

And also make sure that you have configured authenticate your github account into git on your computer. If not you can use ssh connected to github.

**Comment by viandwi24** [comment]
If there is a permission error, make sure that you push to your own repo (after fork). Don't push to this repo, because of course you don't have permission.

And also make sure that you have configured authenticate your github account into git on your computer. If not you can use ssh connected to github.
